scrapper.py

```diff
@@ -25,39 +36,159 @@ class Crawler:
     """
     Crawler implementation
     """
-    def __init__(self, seed_urls, max_articles: int):
-        pass
 
-    def _extract_url(self, article_bs):
-        pass
+    def __init__(self, seed_urls, max_articles: int):
+        self.max_articles = max_articles
+        self.seed_urls = seed_urls
+        self.count_articles = 0
+        self.urls = []
+
+    def _extract_url(self, article_bs):  # I don't understand if I need to collect seed urls (from main to year)
```

Comment:
remove such questionable comments from the code and ask the question in the forum.

comments are needed for describing the code and not you concerns